mantidproject/mantid
# 15748 load nexus logs from entry

LoadNexusLogs now allows an optional entry for an NXentryName. In LoadEventNexus, if you give an NXentryName, the logs are going to be read correctly, instead of being the logs from the first entry.

**To test:**



All unit and system tests should pass. Run LoadEventNexus on some files with multiple NXentries, and check if the corresponding logs are loaded

Fixes #15748.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
